### DISCUSSION TITLE
OS Suggestions

### DISCUSSION THREAD

**Discussion by Cotrox** [zepp-health/zeppos-forum#90] [Developer Feedback]
ZeppOS is a very nice OS but I think there are some things we can work on to make it better!

1. Add support for more emoji. Is not so nice to see that 90% of emoji is not currently working.
2. Add fast access to applications clicking on widgets of the default watchface.
3. Add dot on notification bar if you have unread notifications.
4. Add a different way to navigate to installed application like apple watch do.
5. Add Fast Messages that you can use to send them from the watch when you receive a call that you can't answer.
6. Fix the interval API problem that make setted interval not respect the value you put in!

If you guys have other things to suggest comment there, maybe official developers can use our feedback ^^

**Comment by MyoungXUE** [comment]
Hello, thank you very much for your advice.

Questions 1, 4, 5 I will find the relevant responsible colleagues to answer you.

For question 2, please refer to https://github.com/zepp-health/zeppos-forum/discussions/88

For question 3, please refer to https://github.com/zepp-health/zeppos-forum/discussions/89

As for question 6, can you provide more clues to help us troubleshoot the problem?

**Reply by Cotrox** [reply]
About point 6. I'm sorry if 'm late. I was sure to answered you XD

And i'm also sorry for my bad English.

Anyway... i'm trying to realize a Metronome App for ZeppOS. You can find more info about my project there: https://github.com/orgs/zepp-health/discussions/79

As you can see there is an interval not respected problem. I also have seen the new publication of a similar app just some days ago on the App Store. They have the same problem that I have.

I also want to ask you how can i publish my App on the store because i put it on review (i think) but i nevere receive an answer about the publication from Zepp. 

Thanks as always for your time!

**Comment by raylewisdesign** [comment]
Thank you for your feedback. 
We're pleased to inform you that we'll be adding support for questions 1 and 4 in our upcoming OS 3.0. Regarding question 5, while we've implemented the hang up and reply SMS feature for most Android phones, we regret to inform you that due to iOS limitations, this feature is currently not available for iPhones.

**Reply by Cotrox** [reply]
Is there a release date for the 3.0 version of zepp OS?ù

About iOS do you think it can be possible in the future? For exemple Windows is adding the support for notification read on Windows, calls and messages for iOS. This feature was not possible in past.

**Reply by raylewisdesign** [reply]
The release of version 3.0 is expected around mid-year. Currently, based on the limitations of iOS Bluetooth protocol, the feature you mentioned cannot be supported. However, I can provide your feedback and new information to our development team for further investigation.

**Reply by Cotrox** [reply]
Thank you!

**Comment by Cotrox** [comment]
Hi! I've some new suggestions for you!

- Microsoft To-Do Support to read Microsoft To-Do tasks directly from your watch.
- Yazio Support to read Yazio Data on your Watch or on Zepp App.
- Live Sports support to read your favorite sports results.
- Windows 11 Unlock as Mi Band do with the App: Mi Band Blaze Unlock.

**Reply by raylewisdesign** [reply]
Great suggestion! We'll add it to our Inspiration Pool!

**Reply by raylewisdesign** [reply]
By the way, you might find some surprises in the Zepp App Store.

**Reply by Cotrox** [reply]
Yes, I know the App Store 😍

**Comment by Cotrox** [comment]
Another Suggestion is to add the ability to organize App in 'folders'.

**Reply by MyoungXUE** [reply]
Hi, can you elaborate a bit, I guess you want to use folders to include all the Mini Programs in them, similar to a secondary menu?

**Comment by Cotrox** [comment]
Well yes, you told us that with ZeppOS 3.0 will be add a new way to see all the apps. I think can be a nice thing to add the way to organize your app in different folders directly from watch or from zepp App maybe. In this way we can create a group a mini programs, for exemple a group for sports mini programs, games, tools and other things..

Giuseppe A. Cotroneo
________________________________
Da: Myoung XUE - Zepp Health ***@***.***>
Inviato: Friday, March 10, 2023 3:28:05 AM
A: zepp-health/zeppos-forum ***@***.***>
Cc: Giuseppe Antonino Cotroneo ***@***.***>; Author ***@***.***>
Oggetto: Re: [zepp-health/zeppos-forum] OS Suggestions (Discussion #90)


Hi, can you elaborate a bit, I guess you want to use folders to include all the Mini Programs in them, similar to a secondary menu?

**Comment by Cotrox** [comment]
I have a new suggesstion and it is very **important** for iOS Users.

On iPhone you have to let Zepp App Opened in Background to use 100% of Watch Abilities.

But also Zepp Widgets need the app to be opened to refresh data.

Why don't use Widget as background auto refresh? In this way, people can add widget to let Watch get data without open Zepp App.
